README.md

```diff
@@ -139,6 +139,6 @@ MIX_SENTRY_VUE_DSN=<snip>
 5. **Optimierung (optional)**: `docker-compose run --entrypoint "composer install --optimize-autoloader --no-dev" qualix`
 6. **Auf den Webhost hochladen**: Z.B. mit FTP alles (Ordner und Dateien) ausser .git, cypress, node_modules und tests hochladen
 7. **Mit SSH auf den Server einloggen**, da die folgenden Befehle in der finalen Umgebung ausgeführt werden müssen
-8. **Optimierung (optional)**: `php artisan config:cache && php artisan route:cache`
+8. **Optimierung (optional)**: `php artisan optimize && php artisan view:cache`.
```

Comment:
```suggestion
8. **Optimierung (optional)**: `php artisan config:cache && php artisan route:cache && php artisan view:cache`
```

`artisan optimize` is obsolete since Laravel 5.5, because PHP op-caching is apparently better... https://laravel.com/docs/5.5/upgrade

But `artisan route:cache` is still a thing: https://laravel.com/docs/9.x/routing#route-caching Any specific reason why you removed that? Did it not work?

Also there is `artisan config:cache` which you fix in the other PR. Config, routes and views are the three things that can be cached nowadays: https://laravel.com/docs/9.x/deployment#optimizing-configuration-loading

Reply:
Ah interesting. I just saw the optimize command in the `php artisan` help. It seems to be doing just those three commands… but if it's deprecated I think your proposal is better, as it may avoid confusion.

I'm also going to throw in the link to the docs in the readme :)